thirdeye/thirdeye-frontend/app/pods/components/rootcause-metrics/component.js

```diff
@@ -192,6 +192,43 @@ export default Ember.Component.extend({
     }
   ),
 
+  /**
+   * Urls of associated metric, keyed by metric url
+   * @type {Object} - key is metric urn, and value is an array of objects, each object has a key of the url label,
+   * and value as the url
+   * @example
+   * {
+   *  thirdeye:metric:12345: [],
+   *  thirdeye:metric:23456: [
+   *    {urlLabel: url},
+   *    {urlLabel: url}
+   *  ]
+   * }
+   */
+  links: Ember.computed(
+    'urns',
```

Comment:
add 'entities'

Reply:
`urns` already listens to `entities`, so I figured it would be a duplicate listener if I add `entities`. Thoughts?